src/util.rs

```diff
@@ -52,12 +52,12 @@ impl Drop for TempFifo {
 
 /// Escape Kakoune string wrapped into single quote
 pub fn editor_escape(s: &str) -> String {
-    s.replace("'", "''")
+    s.replace('\'', "''")
```

Comment:
not sure about this. If we want this change it should be in a separate commit, and handle other single-char replace like in `src/language_features/hover.rs` too.

Reply:
Oh yeah, I just naively resolved the clippy fixmes out of habit - I'll clear them out of this

Reply:
Huh, I didn't see clippy complaining. OTOH I do get code actions from rust-analyzer to transform string -> char and vice versa, but those are just assists.

Reply:
It might have been those - i may have just assumed they were from clippy

Reply:
https://rust-lang.github.io/rust-clippy/master/index.html#single_char_pattern
Ah, yes, there it is

Reply:
strange that my clippy doesn't suggest that. Anyway, since it's blessed by clippy the idea seems okay.
If possible we should have a reproducible `cargo clippy` config, not sure how that would work